README.md

```diff
@@ -304,31 +170,37 @@ The following optional dependencies exist:
 ```bash
 apt install \
   build-essential \
-  qt5-default qtmultimedia5-dev libqt5multimedia5-plugins qttools5-dev-tools \
+  qt5-default qttools5-dev-tools \
   libasound2-dev \
   libpulse-dev \
+  librtaudio-dev \ #optionally
+  librtmidi-dev \ #optionally
   libjack-jackd2-dev # for JACK2, or libjack-dev for JACK1
 ```
 
+##### Arch Linux/Manjaro/Artix:
+```bash
+pacman -S \
+  qt5-tools \
+  pulseaudio \
+  jack \ 
+  rtaudio \ #optionally
+  rtmidi \ #optionally
+  alsa-plugins #uncertain about this one though
```

Comment:
Is this something you wanted to verify still?

Reply:
Hmm.. must verify this as well: https://aur.archlinux.org/packages/bambootracker-git/

Reply:
#311 :thinking: